WordPress/gutenberg
# Shortcut: Add Storybook stories

## What?

- Add missing story for[ Shortcut Component](https://github.com/WordPress/gutenberg/tree/trunk/packages/components/src/shortcut)
- Remove `@example` syntax on JSDoc for these components so Storybook can parse properly https://github.com/WordPress/gutenberg/pull/53379#issuecomment-1667472444

## Why?

Missing stories for components has been discussed on https://github.com/WordPress/gutenberg/issues/36128#issuecomment-1650618499

## How?

- Add new /stories folder on /shortcut
- Remove `@example` syntax on the noted components
- About `Text` component: its stories and type migration should be addressed on followed-up PRs

## Testing Instructions




- Checkout this branch
- Run `npm run storybook:dev`
    1. Confirm the shortcut stories at `?path=/docs/components-shortcut--docs` is working
    2. Confirm [these stories](https://github.com/WordPress/gutenberg/pull/53379#issuecomment-1667472444) has the correct JSDoc on storybook

### Testing Instructions for Keyboard


## Screenshots or screencast 
<img width="1448" alt="image" src="https://github.com/WordPress/gutenberg/assets/10071857/8255169c-90cf-46a9-87d8-b50b0faf542e">
